tests/applications/Applications.DicomDirectoryProcessor.Tests/Applications.DicomDirectoryProcessor.Tests.csproj

```diff
@@ -2,6 +2,12 @@
   <PropertyGroup>
     <AssemblyName>Applications.DicomDirectoryProcessor.Tests</AssemblyName>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NUnit.Analyzers">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
```

Comment:
We can add this block once to `tests/Directory.Build.props`, I think

Reply:
Done.